app/template/index.html

```diff
@@ -134,9 +134,22 @@
                            <button class="btn btn--white btn--dropdown selected view">Different Views</button>
```

Comment:
The legend and help information needs to be updated.

Reply:
Will cover this in the next iteration/PR